assets/csidriveroperators/powervs-block/hypershift/mgmt/06_deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: powervs-block-csi-driver-operator
+  namespace: ${CONTROLPLANE_NAMESPACE}
+  annotations:
+    config.openshift.io/inject-proxy: powervs-block-csi-driver-operator
```

Comment:
Can you check if we need this proxy? https://github.com/openshift/cluster-storage-operator/pull/337/files

Reply:
We don't need this proxy, tested without it and it worked fine. 
So removed it.